plugins/goodix-fp/README.md

```diff
@@ -0,0 +1,31 @@
+Goodix Fingerprint Support
+================
+
+Introduction
+------------
+
+The plug used for update firmware for fingerprint sensors.
+
+Firmware Format
+---------------
+
+The daemon will decompress the cabinet archive and extract a firmware blob in
+a packed binary file format.
+
+This plugin supports the following protocol ID:
+
+ * com.goodix.goodixfp
```

Comment:
Considering this is for match on chip, does the match on host use the same flashing protocol?  If not, then I think this protocol should probably use the word "moc" somewhere.

Reply:
right.

Reply:
I' ve changed `goodixfp` to `goodixmoc`

Reply:
Should the plugin be called `goodixmoc` rather than `goodixfp` perhaps?